docker/node/docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
       - '--web.console.libraries=/usr/share/prometheus/console_libraries'
       - '--web.console.templates=/usr/share/prometheus/consoles'
   grafana-ui:
-    image: grafana/grafana:5.0.0
+    image: grafana/grafana:5.2.1
```

Comment:
Are we trying to match the release numbers? I think we bumped it to 5.3 now

Reply:
Correct. Version as of today is 5.2.3

Reply:
You can just remove it and have it grab latest. It all depends how particular you want to be. For grafana v5.2.2 one is the latest just released a few hours ago. v5.2.1 is fine for now. I'll do another update of versions later this week. Maybe I'll remove versioning off. Still working with v4.x grafana vs v5 as I have some items in v4 that are nice but might take a long time to move to v5. I'll have more by the end of the week. This is good for now. It will evolve.